static/src/js/util/request/__tests__/colorMapRequest.test.js

```diff
@@ -0,0 +1,55 @@
+import { v4 as uuidv4 } from 'uuid'
+
+import ColorMapRequest from '../colorMapRequest'
+import Request from '../request'
+
+jest.mock('uuid')
+uuidv4.mockImplementation(() => 'mock-request-id')
+
+beforeEach(() => {
+  jest.restoreAllMocks()
+  jest.clearAllMocks()
+})
+
+describe('ColorMapRequest#constructor', () => {
+  test('sets the default values when authenticated', () => {
+    const token = '123'
+    const request = new ColorMapRequest(token)
+
+    expect(request.authenticated).toBeTruthy()
+    expect(request.authToken).toEqual(token)
+    expect(request.baseUrl).toEqual('http://localhost:3000')
+  })
+})
+
+describe('ColorMap#transformResponse', () => {
```

Comment:
ColorMapRequest does not have a `transformResponse` method, you can leave that test to the class implementing that method